.vsts-ci.yml

```diff
@@ -16,17 +16,19 @@ variables:
     _PublishType: blob
     _SignType: real
 
-phases:
+jobs:
 - template: /eng/build.yml
   parameters:
     agentOs: Windows_NT
-    queue:
+    pool:
       ${{ if eq(variables['System.TeamProject'], 'public') }}:
-        name: dotnet-external-temp
+        name: NetCorePublic-Int-Pool
```

Comment:
This naming scheme is confusing. I was about to say "no, this should be in the public pool". but it is.

Reply:
Good feedback for @mmitche .  Any suggested names?

Reply:
I think avoiding "Internal" and "Int" in public names and likewise "public" and "pub" internal names would help.